.github/workflows/watt-cd.yml

```diff
@@ -63,12 +63,45 @@ jobs:
           exitzeroonchanges: true # 👈 Option to prevent the workflow from failing
           buildscriptname: designsystem:build
 
+  npm_publish:
+    name: Publish to NPM
+    continue-on-error: true
```

Comment:
this might get removed later when the job has been confirmed to be working and stable.